src/components/Table/Table.tsx

```diff
@@ -16,6 +17,9 @@ export interface TableColumnConfiguration {
   header: string; // Column heading
   alignRight?: boolean | undefined; // Align content to the right?
   width?: WidthPercent; // Fixed percentage of table width for column to consume
+  cellWordBreak?: boolean;
+  cellDisableWordWrap?: boolean;
+  headerWordWrap?: boolean;
```

Comment:
Could you please add comments that spell out what each property is meant to achieve?